editor/src/components/canvas/canvas-strategies/strategies/grid-resize-element-strategy.ts

```diff
@@ -38,11 +38,9 @@ export const gridResizeElementStrategy: CanvasStrategyFactory = (
     return null
   }
 
-  const isFillContainer = isFixedHugFillModeApplied(
-    canvasState.startingMetadata,
-    selectedElement,
-    'fill',
-  )
+  const isFillContainer =
```

Comment:
should we rename this variable, or is still sensible?

Reply:
good point, for consistency I renamed it to `isFillOrStretchContainer`